### PR TITLE
Swimming Ducks

### PR DESCRIPTION
**Pull request by HyTurtle**

### CODE REVIEW

File: aaaaAAAA/game.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        super().__init__()
+        self.curtains = Curtains(self)
+        self.curtains.add_scene('swimming_scene', DuckScene())
+        self.curtains.set_scene('swimming_scene')
+        arcade.set_background_color(arcade.color.WARM_BLACK)
+        self.path = []
+
+    def on_key_release(self, symbol: int, modifiers: int) -> None:
+        """When buttons are pressed."""
+        if symbol == 97:
```
Comment (lemonsaurus) [inline aaaaAAAA/game.py:77]: These could probably be better documented. What's 97? Maybe an inline comment, or better yet, an Enum?

File: aaaaAAAA/game.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            if self.curtains.current_scene == self.curtains.scenes['swimming_scene']:
+                self.curtains.current_scene.add_a_ducky()
+        elif symbol == 112:
```
Comment (lemonsaurus) [inline aaaaAAAA/game.py:80]: Same thing here.

File: aaaaAAAA/game.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    def draw(self):
+        """When a thing is drawn."""
+        arcade.start_render()
+        arcade.draw_lrwh_rectangle_textured(
+            0, 0, constants.SCREEN_WIDTH, constants.SCREEN_HEIGHT, self.background
+        )
+        self.ducks.draw()
+
+    @staticmethod
+    def sequence_gen(random: Optional[bool] = False) -> Sequence:
```
Comment (lemonsaurus) [inline aaaaAAAA/game.py:52]: Might be nice with a docstring for this one.

File: aaaaAAAA/game.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        self.animations.fire(ducky, seq)
+        if len(self.ducks) >= DUCKS:
+            arcade.unschedule(self.add_a_ducky)
+
+    def enter_scene(self, previous_scene) -> None:
+        self.animations.fire(self.leader, self.seq)
+        arcade.schedule(self.add_a_ducky, 3)
+
+    def draw(self):
+        """When a thing is drawn."""
```
Comment (lemonsaurus) [inline aaaaAAAA/game.py:44]: Docstrings should be imperative mood. So, "Draw a thing", not "When a thing is drawn".

Also maybe we should say something about what we're drawing.
```suggestion
        """Draw ducks to the screen!"""
```

**Review by lemonsaurus — APPROVED** [review]
Nice. Here's a few suggestions though.

File: aaaaAAAA/game.py (hunk 1 above)
Comment (HyTurtle) [reply aaaaAAAA/game.py:77]: Documented, should I ord('a') as well?

Comment (lemonsaurus) [reply aaaaAAAA/game.py:94]: ord('a') is definitely more readable. Why not?

Comment (lemonsaurus) [reply aaaaAAAA/game.py:94]:
```suggestion
        if symbol == ord('a'):
```

File: aaaaAAAA/game.py (hunk 2 above)
Comment (lemonsaurus) [reply aaaaAAAA/game.py:97]:
```suggestion
        elif symbol == ord('p'):
```

File: aaaaAAAA/game.py [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        'x' to clear the points
+        """
+        if not self.debug:
+            return
+        if symbol == 97:
+            if self.curtains.current_scene == self.curtains.scenes['swimming_scene']:
+                self.curtains.current_scene.add_a_ducky()
+        elif symbol == 112:
+            print(POINTS_HINT)
+        elif symbol == 120:
```
Comment (lemonsaurus) [inline aaaaAAAA/game.py:99]:
```suggestion
        elif symbol == ord('x'):
```

**Review by lemonsaurus — APPROVED** [review]
Let's do this and then merge.

**Pull request merged**